Tests/Data/geometry-hierarchy-map.json

```diff
@@ -0,0 +1,39 @@
+{
+    "acts-geometry-hierarchy-map": {
+        "format-version": 0,
+        "value-identifier": "thing"
+    },
+    "entries": [
+        {
+            "_comment": "global default entry w/o identifier",
```

Comment:
Is the "_" on purpose ? It is also in the docs.
I think having comments is a really nice idea, do you think we could have a way to add them without having to do it by hand ?

Reply:
Json does not support comments. Depending on the parser, they might support custom comment syntax, but the Json standard does not include them. Adding comments as additional entries seems to be a common way to circumvent this. Prefixing with `_` is done to indicate that this is a special entry.

Reply:
I don't mind doing it this way, but I've literally never seen a comment prefixed by `_` like this.

Reply:
I did not come up with this myself, I am pretty sure of that. Let me see if I can find where I took the inspiration from.

Reply:
I make sense, it looks a bit weird but that is fine.